sdk/ml/azure-ai-ml/azure/ai/ml/operations/_schedule_operations.py

```diff
@@ -161,14 +161,14 @@ def get(
     def begin_create_or_update(
         self,
         schedule,
```

Comment:
We seem to be missing a type hint
```suggestion
        schedule: Schedule,
```

Reply:
added. I also added type hints for some other arguments that were missing them